src/tools/compiletest/src/runtest.rs

```diff
@@ -1632,7 +1632,7 @@ impl<'test> TestCx<'test> {
         if self.props.remap_src_base {
             rustc.arg(format!(
                 "--remap-path-prefix={}={}",
-                self.config.src_base.display(),
+                self.config.src_test_suite_root.to_str().unwrap(),
```

Comment:
This was very confusing to me, because this is not the source root, but actually the root of the test suite sources